neuro_config_client/client.py

```diff
@@ -276,6 +279,159 @@ async def remove_storage(
                 raise
         return await self.get_cluster(cluster_name)
 
+    async def get_node_pool(
+        self,
+        cluster_name: str,
+        node_pool_name: str,
+        *,
+        token: str | None = None,
+    ) -> NodePool:
+        assert self._client
+        url = (
+            self._clusters_url
+            / cluster_name
+            / "cloud_prodider"
+            / "node_pools"
+            / node_pool_name
+        )
+
+        headers = self._create_headers(token=token)
+        async with self._client.get(url=url, headers=headers) as responce:
+            responce.raise_for_status()
+            resp_payload = await responce.json()
+            return self._entity_factory.create_node_pool(resp_payload)
+
+    async def get_node_pools(
+        self,
+        cluster_name: str,
+        *,
+        token: str | None = None,
+    ) -> list[NodePool]:
+        assert self._client
+        url = self._clusters_url / cluster_name / "cloud_prodider/node_pools"
+
+        headers = self._create_headers(token=token)
+        async with self._client.get(url=url, headers=headers) as responce:
+            responce.raise_for_status()
+            resp_payload = await responce.json()
+            return [self._entity_factory.create_node_pool(n) for n in resp_payload]
+
+    async def add_node_pool(
+        self,
+        cluster_name: str,
+        node_pool: NodePool,
```

Comment:
You cannot use this model to add/update node pool. It's different for cloud and onprem clusters.

Reply:
same here, thank you for help, pls check [be58386](https://github.com/neuro-inc/neuro-config-client/pull/83/commits/be5838666c221fd64f7f1085e93d2598905e231a)